joomla/joomla-cms
# [4.0] Template Atum - Cleanup and fix Logo Usage

Pull Request for Issue #28826

### Summary of Changes
- Removed unused logos
- Renamed logos to more generic names
- Moved logos to subfolder
- Renamed variables to more recognizeable names
- Changed scss
- Added sizes to svg instead scss
- Changed order of Login input fields 
- Changed Language Strings

### Testing Instructions
- Install the patch
- npm run build:css
- check login logo, logo in headerbar, expand and collapse the sidebar
- upload own logos:
Custom header collapsed Logo recommended max. size 25px x 25px
Custom header expanded Logo recommended max. size 275px x 45 px


- Question: The Logo Variables repeat all over the template files, would it be an idea to have a logos.php file and include it into the severall template files?

- Login Logo appears black for now but only until the PR #28788 will be merged

![grafik](https://user-images.githubusercontent.com/828371/80414299-1af49380-88d1-11ea-9638-0cec30f00457.png)

![grafik](https://user-images.githubusercontent.com/828371/80414388-424b6080-88d1-11ea-9401-5d0325b5444c.png)

![grafik](https://user-images.githubusercontent.com/828371/80414796-df0dfe00-88d1-11ea-85f2-7c8c3cff6478.png)


![grafik](https://user-images.githubusercontent.com/828371/80415669-2648be80-88d3-11ea-8fa7-d2f19bcded04.png)
![grafik](https://user-images.githubusercontent.com/828371/80415702-2f399000-88d3-11ea-9976-6d713b0c6bdb.png)
![grafik](https://user-images.githubusercontent.com/828371/80415719-33fe4400-88d3-11ea-8e2b-23d27b387cd0.png)
